projects/packages/boost-speed-score/tests/bootstrap.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Bootstrap.
+ *
+ * @package automattic/
+ */
+
+/**
+ * Include the composer autoloader.
+ */
+require_once __DIR__ . '/../vendor/autoload.php';
+
+// Additional functions that brain/monkey doesn't currently define.
```

Comment:
If you really intend to use WorDBless, you don't need this but you do need to be calling `\WorDBless\Load::load()`. If you don't intend to use WorDBless, you should remove it from composer.json.

P.S. If you want to use both brain/monkey and WorDBless, your bootstrap should look more like
https://github.com/Automattic/jetpack/blob/e7090fe09442268b1513b625c07fd114f4daf083/projects/packages/partner/tests/php/bootstrap.php#L8-L21